### PR TITLE
Github fetcher binary

### PR DESCRIPTION
**Pull request by apelisse**
This is putting everything together to:
- Read the various configurations (database, github client)
- Fetch issues and issue events in a loop (or do it once, with requested frequency)

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/410)

### CODE REVIEW

File: github-stats/fetcher/fetcher.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+package main
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/golang/glog"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
```
Comment (fejta) [inline github-stats/fetcher/fetcher.go]: What is happening here? Why the _?

Comment (apelisse) [inline github-stats/fetcher/fetcher.go]: Yeah, that's weird. The code is not used per se, but it needs to be included (it's used at run-time)

Comment (spxtr) [inline github-stats/fetcher/fetcher.go]: https://golang.org/ref/spec#Import_declarations

If you need the initialization but nothing else you do this.

**Comment by apelisse** [comment]
@fejta Any comment beside that?

**Comment by fejta** [comment]
Nope, all good otherwise!

**Comment by apelisse** [comment]
Awesome, this is almost over! We just miss a dockerfile and a deployment yaml file now :-)

**Pull request merged**